tests/main/interfaces-iio/task.yaml

```diff
@@ -36,13 +37,14 @@ restore: |
     rm -f /dev/iio:device0
 
 execute: |
-    . $TESTSLIB/dirs.sh
+    #shellcheck source=tests/lib/dirs.sh
+    . "$TESTSLIB"/dirs.sh
 
     if [ "$TRUST_TEST_KEYS" = "false" ]; then
         echo "This test needs test keys to be trusted"
         exit
     fi
-    test "`$SNAP_MOUNT_DIR/bin/iio-consumer.read`" = "iio-0"
+    test "$($SNAP_MOUNT_DIR/bin/iio-consumer.read)" = "iio-0"
 
     $SNAP_MOUNT_DIR/bin/iio-consumer.write "hello"
-    test "`$SNAP_MOUNT_DIR/bin/iio-consumer.read`" = "hello"
+    test "$($SNAP_MOUNT_DIR/bin/iio-consumer.read)" = "hello"
```

Comment:
funny how it hasn't asked you to quote the inner var here

Reply:
That's because shellcheck is aware of `$SNAP_MOUNT_DIR` (as it goes after the sourced files) and that it contains no spaces.